breathe/renderer/sphinxrenderer.py

```diff
@@ -933,8 +1013,8 @@ def render_signature(file_data, doxygen_target, name, kind):
         ("public-slot", "Public Slots"),
         ("signal", "Signals"),
         ("dcop-func", "DCOP Function"),
-        ("property", "Property"),
-        ("event", "Event"),
+        ("property", "Properties"),
+        ("event", "Events"),
```

Comment:
Notes to self: update comment above this section and update table in `documentation/source/file.rst`. (I will handle this)